.kodiak.toml

```diff
@@ -0,0 +1,13 @@
+version = 1
+
+[merge]
```

Comment:
thought (not-blocking): have you given any thought to this config toggle: https://kodiakhq.com/docs/config-reference#mergeprioritize_ready_to_merge

While this does some "unfairness", this likely saves build cycles by merging in the logical fastest way and not blocking a chain of PRs on a fail for whatever is ahead in the queue

Reply:
I don't necessarily have a good sense of how they actually deal with the queue yet. It sounds like they don't just keep rebasing and rebuilding everything, they only do it one at a time based on whats next. So hopefully they shouldn't be spinning up CI all the time for no reason.

If that's true, skipping the queue might actually introduce more CI usage becuase something in the queue that is currently building, might then have to be rebuilt if something jumps ahead of it.

Lets keep it in our back pocket. There are likely a lot of config options that could help optimise our queue once we get a good picture on how it actually works for us.